ospf6d/ospf6_intra.c

```diff
@@ -1571,12 +1571,15 @@ void ospf6_intra_prefix_route_ecmp_path(struct ospf6_area *oa,
 					ifp = if_lookup_prefix(
 						&old_route->prefix,
 						oa->ospf6->vrf_id);
-					if (ifp)
-						ospf6_route_add_nexthop(
-								old_route,
```

Comment:
is it a case of P2P interface? Can you please share example of config that you used to recreate this issue?